app/src/main/java/com/paypal/psix/models/Event.java

```diff
@@ -80,6 +80,8 @@ public Event setup() {
         return this;
     }
 
+    public int getNumberOfInvitees() { return this.rsvps().size() - 1; }
```

Comment:
I assume this deduction is for the host, but I'm not sure it's needed at this point. An event host should be able to see the number of all of the people that are going to go to an event. And it might be confusing if he sees a different number in the app and on Facebook. For calculating payments we might want to deduce one, but not in general.